### ISSUE TITLE
On follow, deliver last few posts

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy-fed#742]
When we get a follow, should we deliver the last few posts from the followee to the follower? Seems like it'd be nice UX. .Those posts' published/updated dates will be intact, so they should merge into the appropriate places in the follower's feed and not all show up at the top.

**Comment by WisTex** [comment]
Hubzilla grabs the last 10 posts when you follow someone (if their platform supports it). It is a really nice feature. I think it should be adopted, where technically feasible. 

It would also be nice if you can detect threaded conversations, and get the whole conversation. (Only platforms that have threaded conversations would support this.)

**Comment by Tamschi** [comment]
This probably used to work on ActivityPub, but now doesn't according to https://github.com/snarfed/bridgy-fed/issues/1248#issuecomment-2289615188.
On the Bluesky side, something seems to be amiss to some extent too, as atproto-browser ["🚨 Failed to fetch collections."](https://atproto-browser.vercel.app/at/Tamschi.mastodon.gamedev.place.ap.brid.gy)

**Comment by snarfed** [comment]
Right, some fediverse servers backfill from outbox, which I disabled in #1248, but not all. I was thinking about proactively delivering historical posts for the servers that don't backfill.

"Failed to fetch collections" is probably because our `com.atproto.repo.describeRepo` implementation is incomplete right now: https://github.com/snarfed/arroba/issues/33 . Worth fixing, but probably unrelated here.
